tvb_contrib/tvb/contrib/tests/model/model_test.py

```diff
@@ -41,12 +41,14 @@
 from tvb.contrib.simulator.models.morris_lecar import MorrisLecar
 from tvb.contrib.simulator.models.wong_wang import WongWang
 from tvb.simulator.integrators import HeunDeterministic
-from tvb.tests.framework.core.base_testcase import BaseTestCase
+# from tvb.tests.framework.core.base_testcase import BaseTestCase
```

Comment:
This commented line could be removed directly.
We should not import from tvb-framework in tvb-contriv